app/views/layouts/_seo.html.erb

```diff
@@ -0,0 +1,50 @@
+<%# This belongs inside the <head> tag %>
+
+<link rel="publisher" href="https://plus.google.com/u/0/+IndentLabs" />
```

Comment:
I think we want to drop the /u/0 here -- it's an identifier for which Google account _you_ are logged into, rather than actually part of the URL.